experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -177,13 +165,13 @@ function main()
     CLIMA.init()
 
     # Driver configuration parameters
-    FT = Float32                        # floating type precision
-    poly_order = 5                      # discontinuous Galerkin polynomial order
-    n_horz = 5                          # horizontal element number
-    n_vert = 5                          # vertical element number
-    days = 100                          # experiment day number
-    timestart = FT(0)                   # start time (seconds)
-    timeend = FT(days * 24 * 60 * 60)   # end time (seconds)
+    FT = Float32                             # floating type precision
+    poly_order = 5                           # discontinuous Galerkin polynomial order
+    n_horz = 5                               # horizontal element number
+    n_vert = 5                               # vertical element number
+    n_days = 365.26                          # experiment day number
```

Comment:
Really :). This choice will seem quite odd to those unitiated to the finer points of celestial mechanics. Maybe 365 will do?

Reply:
We should use the `day` method in CLIMAParameters :)